Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -189,43 +272,50 @@ lemma normEDS'_even (m : ℕ) : normEDS' b c d (2 * (m + 3)) * b =
     ring1
 
 /-- The canonical example of a normalised EDS `W : ℤ → R`,
-with initial values `W(0) = 0`, `W(1) = 1`, `W(2) = b`, `W(3) = c`, and `W(4) = b * d`.
+with initial values `W(0) = 0`, `W(1) = 1`, `W(2) = b`, `W(3) = c`, and `W(4) = d * b`.
 
 This extends `normEDS'` by defining its values at negative integers. -/
-def normEDS (n : ℤ) : R := n.sign * normEDS' b c d n.natAbs
+def normEDS (n : ℤ) : R :=
+  n.sign * normEDS' b c d n.natAbs
```

Comment:
Can't you define
```
def normEDS (n : ℕ) : R :=
  preNormEDS (b ^ 4) c d n * if Even n then b else 1
```
and remove `normEDS'` altogether? You can always recover `normEDS' n` as `normEDS (ofNat n)`.